packages/server/src/database/knex/migrations/20200425004150_create-sessions.js

```diff
@@ -0,0 +1,18 @@
+
+exports.up = function(knex) {
+  return knex.schema.dropTableIfExists('Sessions').createTable('Sessions', function (table) {
+    table.increments('id')
+    table.string('sid').unique()
+    table.jsonb('data')
+    table.integer('adminId')
+    table.foreign('adminId').references('Admins.id').onDelete('CASCADE')
+    table.integer('userId')
+    table.foreign('userId').references('Users.id').onDelete('CASCADE')
```

Comment:
Just curious, this work as nullable foreign keys right?

Reply:
hmm yes I didn't add not nullable for columns.

Reply:
I've added nullable constraints in all migrations where necessary so it's more clear

Reply:
Also added raw constraint below that either adminId or userId has to be not null.